src/components/button_panel.js

```diff
@@ -0,0 +1,26 @@
+import Button from './button';
+
+const buttons = [
+  ['AC', '+/-', '%', '÷'],
+  ['7', '8', '9', 'X'],
+  ['4', '5', '6', '-'],
+  ['1', '2', '3', '+'],
+  ['0', '.', '='],
+];
+const buttonsNew = [];
+const ButtonPanel = () => {
+  for (let i = 0; i < buttons.length; i += 1) {
+    buttonsNew.push(
+      <div key={`group + ${i}`}>
+        {
+          buttons[i].map(name => <Button key={name} name={name} />)
+        }
+      </div>,
+    );
+  }
+  return (
+    buttonsNew
+  );
+};
```

Comment:
- Nice idea grouping buttons in this way and mapping through them. It's concise and neat.